app/Picture.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PIL import Image
+
+if TYPE_CHECKING:
```

Comment:
Why is this conditional import required? For non-obvious stuff like this, please write a comment for future reference.